test/acceptance/behave/components/ngsiv2/entities/list_entities/list_entities_qp_III.feature

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think in other similar cases we use arabic numbers 1, 2, 3, etc. instead of romans I, II and III. But probably it doesn't worth to change the branch such minor thing. Up to you (in the case we prefer using roman number, mark this comment as NTC).